x-pack/plugins/alerting/server/alert/create_alert_factory.ts

```diff
@@ -93,7 +93,7 @@ export function createAlertFactory<
 
       if (numAlertsCreated++ >= maxAlerts) {
         hasReachedAlertLimit = true;
-        throw new Error(`Rule reported more than ${maxAlerts} alerts.`);
+        throw new MaxAlertsReachedError(`Rule reported more than ${maxAlerts} alerts.`);
```

Comment:
This will be caught in a toast?

Reply:
i have updated the approach so this change isn't valid anymore.